compatibility/bazel_tools/testing.bzl

```diff
@@ -675,6 +675,17 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        # Contract ID tests are governed by feature descriptors.
+        "platform_ranges": [
+            {
+                "end": "2.0.0-snapshot.20220110.8812.1",
```

Comment:
Just to make sure, `ContractIdIT` is not going to be run with any `ledger-api-test-tool` version against platform versions older than the next snapshot. Is that what you want?

Reply:
Yes, this reproduces the old behavior, where these tests were optional and therefore not run.